### DISCUSSION TITLE
Can TwoSampleMR perform MR analysis in data obtained from a single cohort ?

### DISCUSSION THREAD

**Discussion by zxwang2019** [MRCIEU/TwoSampleMR#256] [Q&A] [answered]
TwoSampleMR is great and easy to perform MR analysis. However, as its name implies, it applies to the samples from two independent cohort. I wonder if the TwoSampleMR could perform **one-sample** MR analysis or some way to correct for the within-sample correlation?

**Comment by explodecomputer** [comment, marked as answer]
We're working on a paper that thinks about this issue. If there is sample overlap then weak instruments will lead to bias in the direction of the confounder, which is the main reason people try to avoid overlapping samples. But if you have a set of strong instruments then you will probably see very little bias. False discovery rates don't appear to be strongly affected by this either, so it's perhaps not too big a problem. But run sensitivity analyses using only very strong instruments (e.g. with p << 5e-8) to examine how big an issue the sample overlap is likely to be

**Reply by zxwang2019** [reply]
Thank you for your reply！I expect your article to be published.
